manifests/params.pp

```diff
@@ -1,19 +1,34 @@
 # == Class: contrail::params
 #
 class contrail::params(
-  $analytics         = {},
-  $analyticsdatabase = {},
-  $config            = {},
-  $control           = {},
-  $database          = {},
-  $webui             = {},
-){
-
+  $analytics                  = {},
```

Comment:
As discussed earlier, please confirm that these params would be used only for now. Before we GA, the idea is to expose all variables in user-facing tripleo-heat-templates.